File: modul 4/soal 2/PRAK302-2210817220034-Wanda Karliyanti.c
#include<stdio.h>

main(){
    for(int z=1;z<=5;z++){
int nilai;
scanf("%d", &nilai);

if(nilai >= 80){
    printf("A");}

else if(nilai >= 70 && nilai <= 79){
    printf("B");}

else if(nilai >= 60 && nilai <= 69){
    printf("C");}

else if(nilai >= 50 && nilai <= 59){
    printf("D");}

else{printf("E");}

}
}

File: modul 4/soal 2/PRAK302-2210817220034-Wanda Karliyanti.py
for z in range(5):
    nilai=int(input())
    if(nilai >= 80):
        print("A\n")
    elif(nilai >= 70 and nilai <= 79):
       print("B\n")
    elif(nilai >= 60 and nilai <= 69):
        print("C\n")
    elif(nilai >= 50 and nilai <= 59):
        print("D\n")
    else:
        print("E\n")